content/10.zk-stack/20.running-a-zk-chain/80.validium.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Validium in ZK Stack
+description: Dive deeper into the ZK Stack's Validiums, learn how to operate and configure them.
+---
+
+This section provides an in-depth exploration of Validiums in ZK Stack. For more general overview of Validiums,
+see [here](../05.concepts/60.data-availability/20.validiums.md).
+
+## What kinds of Validium are there
+
+We define 3 different kinds of Validiums based on the level of data availability they provide:
+
+- **Stage 0** - the simplest type of Validium that only stores the pubdata in the database of the node(s) running the
+  chain
+- **Stage 1** - a Validium that only sends the data to the DA layer, but doesn’t verify its inclusion
+- **Stage 2** - a Validium that sends the data to the DA layer, and also verifies its inclusion on L1 either by using
+  the verification bridges or zk-proofs directly.
+
+There are multiple DA layers that we are going to integrate with, the first on the list are: Avail, Celestia, EigenDA.
+
+## Server-related details
+
+Getting a bit deeper into the technicalities, the process of persisting the pubdata in the different DA layers is
+handled by two components: DA dispatcher and DA client:
+
+- DA client’s role is to abstract the access to the DA layer and generalize it to 2 functions: `dispatch_blob`
+  and `get_inclusion_data`. Clients are a part of the `zksync-era` repository, see the
+  code [here](https://github.com/matter-labs/zksync-era/tree/main/core/node/da_clients/src).
+- DA dispatcher is responsible for periodically fetching the pubdata from the database, and calling the DA client either
+  to dispatch a blob or get an inclusion proofs. It also handles retries in case the client function call failed with a
+  retryable error.
+
+The DA related information is stored in the `data_availability` table in the database, Validium’s new batches can’t be
+committed if their inclusion data in that table is NULL.
+
+**Configuration**
+
+The Validium chain is required to have the following configuration:
+
+1. Set a pubdata sending mode [general.yaml]
+    - `pubdata_sending_mode: CUSTOM`
+2. Configure DA dispatcher (reference values below) [general.yaml]
+
+    ```yaml
+    da_dispatcher:
+      polling_interval_ms: 5000
+      max_rows_to_dispatch: 100
+      max_retries: 5
+    ```
+
+3. Add DA client’s config and secrets, see FAQ for details.
+
+All the configs can also be set using environment variables instead of `.yaml` files, but it is less preferred.
+
+## Smart contracts
+
+**⚠️ DA inclusion verification contracts are only going to be available in ZK Gateway release**
```

Comment:
```suggestion
::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
DA inclusion verification contracts are only going to be available in ZK Gateway release.
::
```
Also, can we add more context about when this will happen?